### PR TITLE
NN-2754 pathfinder links

### PR DESCRIPTION
**Pull request by JamieDraperUK**
Screenshot 2020-07-09 at 12 37 14
Screenshot 2020-07-09 at 12 38 33

### CODE REVIEW

File: backend/controllers/prisonerProfile/prisonerAlerts.js [hunk 1]
```diff
@@ -117,6 +117,7 @@ module.exports = ({
       canUpdateAlerts,
     })
   } catch (error) {
+    console.error({ error })
```
Comment (jsrobertson) [inline backend/controllers/prisonerProfile/prisonerAlerts.js]: Do we want this?

Comment (JamieDraperUK) [reply backend/controllers/prisonerProfile/prisonerAlerts.js]: Removed 👍

File: sass/_govuk-overrides.scss [hunk 2]
```diff
@@ -69,3 +69,11 @@
 .govuk-warning-text__icon {
   box-sizing: content-box;
 }
+
+.header-with-link {
+  h1 {
+    display: inline;
+    margin-right: 15px;
```
Comment (jsrobertson) [inline sass/_govuk-overrides.scss:76]:
```suggestion
    margin-right: govuk-spacing(3);
```

File: sass/_govuk-overrides.scss [hunk 3]
```diff
@@ -69,3 +69,11 @@
...
+  }
+  margin-bottom: 30px;
```
Comment (jsrobertson) [inline sass/_govuk-overrides.scss:78]:
```suggestion
  margin-bottom: govuk-spacing(6);
```

File: sass/_govuk-overrides.scss (hunk 2 above)
Comment (JamieDraperUK) [reply sass/_govuk-overrides.scss]: Cheers @jsrobertson

**Pull request merged**